lib/index.js

```diff
@@ -16,17 +17,19 @@ module.exports = (pluginConfig, assetManager, options) => {
 };
 
 function buildConfig(pluginConfig, assetManager, { browsers, compact, sourcemaps }) {
-	return pluginConfig.map(bundleConfig => {
-		let inputFileName = assetManager.resolvePath(bundleConfig.source);
-		let target = assetManager.resolvePath(bundleConfig.target, {
-			enforceRelative: true
-		});
+	return pluginConfig.map(({ source, target, browserslist, fingerprint }) => {
+		if(!source || !target) {
+			let setting = source ? "target" : "source";
+			abort(`ERROR: missing ${repr(setting, false)} configuration`);
+		}
+
+		let inputFileName = assetManager.resolvePath(source);
```

Comment:
Why is this not `enforceRelative`?

Reply:
It should be 👍 Please adjust it.

Reply:
I'm not sure anymore. I thought we said this is only temporary, and this time is already gone?

Reply:
Yeah, you're right: 
https://github.com/faucet-pipeline/faucet-pipeline-js/blob/b63dc70a52d1fd8eeedc32727a418bf7e0a2bce1/lib/index.js#L66
So never mind.